packages/types/src/extrinsic/types.ts

```diff
@@ -102,5 +104,6 @@ declare module '@polkadot/types/interfaceRegistry' {
     ChargeTransactionPayment;
     FeeExchange: FeeExchange;
     'Option<FeeExchange>': Option<FeeExchange>;
+    'Vec<(T::AssetId, AssetInfo)>': Vec<ITuple<[AssetId, AssetInfo]>>;
```

Comment:
Could be nice to alias `(T::AssetId, AssetInfo)` as `AssetMeta` or something like that so the type carries more meaning